bats/tests/helpers/paths.bash

```diff
@@ -48,6 +48,13 @@ set_path_resources() {
     if [ ! -d "$PATH_RESOURCES" ]; then
         fatal "App resource directory '$PATH_RESOURCES' does not exist"
     fi
+
+    # Ensure we're bundled executables for sub-processes
+    # (e.g. credential helpers).
+    export PATH="$PATH_RESOURCES/$OS/bin:$PATH"
```

Comment:
I think this code should be combined with https://github.com/rancher-sandbox/rancher-desktop/blob/e73d3ca52e6c85ed5cc51c0feb2d343a013188c0/bats/tests/helpers/load.bash#L74-L75

Modifying `PATH` in 2 locations in different helpers is confusing.